src/stories/containers/Finances/components/HeaderTable/FinancesTable.tsx

```diff
@@ -0,0 +1,131 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
Please, avoid to disable globally eslint, if it is needed to disable a var due this is WIP, do it online, so we remember to delete this after the implementation is complete

Reply:
you are right , done